_maps/configs/pirate_tortuga.json

```diff
@@ -0,0 +1,46 @@
+{
+	"$schema": "https://raw.githubusercontent.com/shiptest-ss13/Shiptest/master/_maps/ship_config_schema.json",
+	"map_name": "Tortuga-class Converted Cargo Freighter",
+	"prefix": "ISV",
+	"namelists": [
+		"PIRATES",
+		"BRITISH_NAVY"
+	],
+	"map_short_name": "Tortuga-class",
+	"map_path": "_maps/shuttles/pirate/pirate_tortuga.dmm",
+	"description": "Originally a cargo transport vessel, the Tortuga class is an improvised combat vessel under the ranks of the Frontiersmen fleet. Clever field modifications include the conversion of the cargo bay to a hangar, and additional padding around the ship in the form of asteroid rocks. It serves a dual purpose, both as a capable ship for establishing a military enclave in a sector, and as a sturdy vessel for reconnaissance. The nimble Brawler-class subshuttle of the Tortuga is offset by its very low overall mobility and loss of operation if the subshuttle were to be lost.",
```

Comment:
"Originally a cargo transport vessel, the Tortuga is an improvised combat vessel used by the Frontiersmen fleet. Civilian transports are modified with shuttle handling facilities, and asteroid rubble is belted to the exterior of the hull as a form of cheap improvised armor and concealment. The Tortuga serves a dual purpose as a reconnaissance element and a vessel capable of establishing forward bases within a sector. While the Brawler-class shuttle carried by the Tortuga offers a surprising degree of speed and range to its boarding parties, the Tortuga itself is sluggish and nearly helpless without its shuttle."

Several edits for clarity and grammar.